packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -167,6 +167,37 @@ exports.createResolvers = ({ createResolvers }) => {
         },
       },
     },
+    Query: {
```

Comment:
should this be conditional based on `GATSBY_EXPERIMENTAL_NO_PAGE_NODES` ?

Reply:
it is — since the rebuild-schema-with-site-pages just overwrites it. But I'll add the conditional just so it's clear